picard/acousticbrainz/__init__.py

```diff
@@ -71,14 +74,15 @@ def ab_setup_extractor(tagger, config=None):
         version = check_extractor_version(acousticbrainz_extractor, tagger)
         if version:
             config.setting["use_acousticbrainz"] = True
-            config.setting["acousticbrainz_extractor_version"] = version
-            config.setting["acousticbrainz_extractor_sha"] = sha
+            ACOUSTICBRAINZ_EXTRACTOR_VERSION = version
+            ACOUSTICBRAINZ_EXTRACTOR_SHA = sha

```

Comment:
Since `ACOUSTICBRAINZ_EXTRACTOR_SHA` is dynamically set, please use lowercased name and since it's a global for internal use (I don't we meant to export it outside this module), underscorde prefix makes sense: `_extractor_sha`

Reply:
Done. o>